dev-qt/qttools/qttools-6.3.0.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qt6-build
+
+DESCRIPTION="Qt Tools"
```

Comment:
TBH I'm not super happy about a monolithic qttools ebuild... most tools in this repo used to be independent and unrelated to each other. Has that changed? how difficult would it be to split them?

Reply:
USE-flags added.

Reply:
>  I'm not sure how to handle file collisions and preserve previously installed tools

not following, file collisions with what? Qt5? why would splitting the package change anything in this regard?

Reply:
> > I'm not sure how to handle file collisions and preserve previously installed tools
> 
> not following, file collisions with what? Qt5? why would splitting the package change anything in this regard?

Perhaps I'm missing something obvious, but let's say we install only assistant, and none of the other tools, setting them OFF, and then install some other tool(s) after that.  I'm thinking we'd need to have a way to retain that assistant install and not install some of the duplicate files that would be installed with all the tools, similar to how we do with Qt5.  I can't quite think of how to do that with CMake.

Reply:
A typical workaround for that situation is to create a `qttools-common` package or similar that installs all common files. We did not have this problem with Qt5's tools though, there were no common files.

Reply:
Right, that did come to mind.  Will look into it as/when time allows.